lib/license_finder/platform.rb

```diff
@@ -1,11 +1,11 @@
 module LicenseFinder
   module Platform
     def self.darwin?
-      RUBY_PLATFORM =~ /darwin/
+      !(RUBY_PLATFORM =~ /darwin/).nil?
     end
 
     def self.windows?
-      RUBY_PLATFORM =~ /mswin|cygwin|mingw/
+      !(RUBY_PLATFORM =~ /mswin|cygwin|mingw/).nil?
```

Comment:
Oh, and if you're specifically asking about the `RUBY_PLATFORM` checks, please again see the commit message (not the PR description):

> Otherwise the offset of the match, if any, or nil would be returned.

I.e. these methods did not really return a boolean.